alembic/env.py

```
@@ -0,0 +1,93 @@
+    if os.environ.get("DATABASE_URL"):
+        configuration["sqlalchemy.url"] = os.environ.get("DATABASE_URL")
+    
+    connectable = engine_from_config(
+        configuration,
+        prefix="sqlalchemy.",
+        poolclass=pool.NullPool,
+    )
+
+    with connectable.connect() as connection:
+        context.configure(
+            connection=connection, target_metadata=target_metadata
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)